### PR TITLE
Warp Rendering

### PR DESCRIPTION
**Pull request by Dinty1**
- Disabled by default
- Activated using lightning button
- Warp data is pulled server-side because CORS and also it's probably pretty intensive. I'll push an action directly to main for auto updating once this is merged
- This switches the map to prefer rendering on a canvas for better performance (haven't seen anything broken yet)
- I know very little about frontend so it's possible I broke something important

### CODE REVIEW

File: warps.json
Comment (Scarycrumb45) [inline warps.json:1]: this file looks pretty empty. did you mean to have some data in here?

**Review by Scarycrumb45 — APPROVED** [review]
lgtm
shaun-the-sheep-movie-ok-gif

File: warps.json
Comment (Scarycrumb45) [reply warps.json:1]: also possible there is data and github is just displaying it incorrectly

Comment (Dinty1) [reply warps.json:1]: It definitely exists. Probably just github weirdness :shrug:
https://raw.githubusercontent.com/MRT-Map/map/warps/warps.json

**Pull request merged**